custom_components/mail_and_packages/const.py

```diff
@@ -582,6 +595,25 @@
         entity_category=EntityCategory.DIAGNOSTIC,
         device_class=SensorDeviceClass.TIMESTAMP,
     ),
+    #eBay
+    "ebay_delivering": SensorEntityDescription(
```

Comment:
Since eBay doesn't actually deliver packages we wouldn't add a "delivering" sensor.